logger/src/main/java/com/orhanobut/logger/MultiLogAdapter.java

```diff
@@ -0,0 +1,60 @@
+package com.orhanobut.logger;
+
+import java.util.List;
+
+/**
+ * Created by ronaldo pace on 1/5/17. https://github.com/budius
+ * Simple LogAdapter implementation that re-distributes the messages to other implementations.
+ */
+public class MultiLogAdapter implements LogAdapter {
```

Comment:
I like the idea of having multiple adapters but instead of another wrapper adapter implementation, I'd introduce a new adapter method in Logger
```
Logger.init()
  .addLogAdapter(new AndroidLogAdapter())
  .addLogAdapter(new AndroidFileLogAdapter())
```

This would be more intuitive. We can still keep the old one as deprecated to provide backward compability and put deprecated annotation